addons/mail/wizard/mail_activity_schedule_views.xml

```diff
@@ -13,39 +13,55 @@
+                            <group colspan="4">
+                                <field name="plan_date" placeholder="Default deadline for the activities..." string="Due Date"/>
                                 <field name="plan_on_demand_user_id" widget="many2one_avatar_user"
-                                       invisible="not plan_has_user_on_demand"/>
+                                       invisible="not plan_has_user_on_demand" string="Responsible"/>
                             </group>
-                            <group>
-                                <label for="plan_summary" class="o_form_label"
-                                       string="Plan summary" colspan="2"/>
-                                <field name="plan_summary" colspan="2" nolabel="1" class="text-muted"/>
+                            <group colspan="8" class="pe-0">
+                                <label for="plan_summary" class="o_form_label mb-n2"
+                                       string="Plan Summary" colspan="2"/>
+                                <field name="plan_summary" colspan="2" nolabel="1" class="text-muted mt-n2 small">
+                                    <list edit="0" no_open="1" class="table-borderless">
+                                        <field name="line" nolabel="1"/>
+                                        <field name="summary_user_id" nolabel="1" width="30px" widget="many2one_avatar_user_noname"/>
+                                        <field name="summary_date_deadline" nolabel="1"/>
+                                        <field name="depth" column_invisible="1" nolabel="1"/>
+                                    </list>
+                                </field>
                             </group>
                         </group>
                         <group invisible="plan_id">
-                            <group>
-                                <field name="res_model" invisible="1"/>
-                                <field name="activity_type_id"
-                                       required="not plan_id"
-                                       options="{'no_create': True, 'no_open': True}"/>
-                                <field name="summary" placeholder="e.g. Discuss proposal"/>
+                            <field name="activity_type_id"
+                                    required="not plan_id"
+                                    invisible="plan_id"
+                                    widget="selection_badge_icons"
+                                    iconField="icon"
+                                    nolabel="1"
+                                    />
+                            <field name="summary" placeholder="e.g. Discuss Proposal" class="fs-3 pb-2"/>
+                            <group colspan="12" invisible="res_ids">
+                                <group col="4">
+                                    <field name="res_model_id" 
```

Comment:
res_model_id is not accessible to standard users, hence you cannot use it, you have to use res_model (I know, annoying, but that's due to security limitation)